src/10-deep-strong-coupling.md

```diff
@@ -0,0 +1,1132 @@
+
+Now we're going to consider the deep strong coupling regime of the Rabi model.
+<!-- #endregion -->
+
+## 10.3 Vacuum field deep strong coupling ($n=0$)
+
+
+Much like the classical pendulum, the dynamics of the the Rabi model depend on the relative sizes of the different frequencies/energies in the system, which are represented by $\Delta E, \hbar\omega, U$ in the Hamilonian:
+
+
+$$H = \frac{\Delta E}{2} \sigma_z + \hbar\omega\left(a^{\dagger}a +\frac{1}{2}\right) + U\left( a^{\dagger} + a \right)(\sigma_+ + \sigma_-)$$
+
+
+The deep strong coupling regime, as first explored by [Casanova et al. in 2010](http://dx.doi.org/10.1103/PhysRevLett.105.263603), is then defined as:
+
+$$
+\Delta E , \hbar \omega \lesssim U
+$$
+
+Casanova first focussed on the case when the TLS is degenerate, so that $\Delta E=0$. We'll follow their lead in order to reduce complexity in these initial stages of exploration. In this case, the condition for deep strong coupling reduces to:
+
+$$
+1 \lesssim \frac{U}{\hbar\omega}
+$$
+
+Before we start simulating, let's see if we can predict what will happen.
+
+
+Ideally, we'd like to appeal to our classical pendulum picture, but this is challenging because the number of states (aka quantum pendulums) that are involved. Each state of the system is described by combining the two states of the TLS (denoted $\pm$) with the boson states (denoted by the number of quanta $n$). There are an infinite number of such states $|n, \pm\rangle$.
+
+Thnking about infinity is hard, so let's begin with thinking about two states and see how far we get.
+
+Let's imagine we start the system in the ground state of the TLS and in vacuum - in other words $\psi_0 = |0,-\rangle$. Because of parity conservation, we know that this state is only coupled to states that simultaneously flip the TLS and shift the boson number by one. The only state that satisfies this is $|1,+\rangle$. The two states have different energies/frequencies from one another and so the pendulum set-up looks like this.
+
+![](./img/pendulums-dsc-01.png)
+
+
+If the pendulums were weakly coupled, then we'd not expect the second pendulum to move much at all because it's natural frequency is very different from the first. For pendulums that are coupled in a deep strong way, we've already seen that one displaced pendulum will pull the other one out of equilibrium even when the natural frequencies are highly mismatched. 
+
+What we can predict for the quantum system is that the vacuum state won't be stable. We're not going to be able to initialise the vacuum state and have it just sit there - other states (with potentially very different energies) are likely to get occupied.
```

Comment:
What other states? Of other TLSs? Or states of the vacuum?

Reply:
`For the quantum system, we can therefore predict that the vacuum state $|0,-\rangle$ won't be stable. We're not going to be able to initialise the vacuum state and have it just sit there - other states with very different energies (in this case the only other state is $|1,+\rangle$) are likely to get occupied.`

How about this?

Reply:
Yes that makes sense to me now